salt/utils/sanitisers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
This module is named using the international/British spelling of "sanitize".

We should either use all international spellings, or all "U.S." spellings, for consistency. I personally don't have a strong opinion either way, but whatever we decide, we should be consistent.